polkadot-parachains/parachains-common/Cargo.toml

```diff
@@ -20,6 +20,7 @@ sp-io = { git = 'https://github.com/paritytech/substrate', branch = "master", de
 frame-executive = { git = 'https://github.com/paritytech/substrate', branch = "master", default-features = false }
 frame-support = { git = 'https://github.com/paritytech/substrate', branch = "master", default-features = false }
 frame-system = { git = 'https://github.com/paritytech/substrate', branch = "master", default-features = false }
+pallet-asset-tx-payment = { git = 'https://github.com/paritytech/substrate', branch = "master", default-features = false }
```

Comment:
Point the branch to [your PR branch here](https://github.com/paritytech/substrate/pull/10127) so that our CI can pick up the correct branch to test your substrate PR against.

Reply:
The only way I managed to make it work was via patches. This made this repo pass the checks, but it's still failing the substrate repo check, would you have an idea?
Also, technically speaking, this PR does not have to be a companion for the substrate one, because you could merge the substrate PR and that would not break cumulus, it would just mean the pallet is defined again in cumulus.

Reply:
patching is fine/intended for this
with this PR green I guess I'd be fine just marking it in Substrate as related and get the CI green and the PR merged